src/simulation/test/LoadGeneratorTests.cpp

```diff
@@ -339,13 +376,13 @@ TEST_CASE("generate soroban load", "[loadgen][soroban]")
         }
     }
 
-    // Manually override network settings for invoke load gen
-    for (auto node : nodes)
-    {
-        overrideSorobanNetworkConfigForTest(*node);
-        modifySorobanNetworkConfig(*node, [&](SorobanNetworkConfig& cfg) {
+    // Override network settings for invoke load gen
+    upgradeSorobanNetworkConfig(
+        [&](SorobanNetworkConfig& cfg) {
+            setSorobanNetworkConfigForTest(cfg);
+
             // Entries should never expire
-            cfg.mStateArchivalSettings.maxEntryTTL = 1'000'000;
+            cfg.mStateArchivalSettings.maxEntryTTL = 2'000'000;
```

Comment:
Why this change?

Reply:
`SorobanNetworkConfig::isValidConfigSettingEntry` [checks that ](https://github.com/stellar/stellar-core/blob/master/src/ledger/NetworkConfig.cpp#L799) `cfg.stateArchivalSettings().maxEntryTTL >= MinimumSorobanNetworkConfig::MAXIMUM_ENTRY_LIFETIME`, and `MAXIMUM_ENTRY_LIFETIME = 1'054'080;`, so 1 million is an invalid setting. The previously used ltx-based `modifySorobanNetworkConfig` function bypasses the validity check.